diff --git a/node_modules/@chakra-ui/toast/dist/esm/use-toast.js b/node_modules/@chakra-ui/toast/dist/esm/use-toast.js
index a6ae100..11d9602 100644
--- a/node_modules/@chakra-ui/toast/dist/esm/use-toast.js
+++ b/node_modules/@chakra-ui/toast/dist/esm/use-toast.js
@@ -29,7 +29,8 @@ var Toast = props => {
     textAlign: "start",
     width: "auto"
   }, /*#__PURE__*/React.createElement(AlertIcon, null), /*#__PURE__*/React.createElement(chakra.div, {
-    flex: "1"
+    flex: "1",
+    maxWidth: "100%"
   }, title && /*#__PURE__*/React.createElement(AlertTitle, null, title), description && /*#__PURE__*/React.createElement(AlertDescription, {
     display: "block"
   }, description)), isClosable && /*#__PURE__*/React.createElement(CloseButton, {
