diff --git a/node_modules/react-mde/lib/definitions/types/CommandOptions.d.ts b/node_modules/react-mde/lib/definitions/types/CommandOptions.d.ts
index 095df80..a9cf32b 100644
--- a/node_modules/react-mde/lib/definitions/types/CommandOptions.d.ts
+++ b/node_modules/react-mde/lib/definitions/types/CommandOptions.d.ts
@@ -16,4 +16,9 @@ export interface TextApi {
      * @param selection
      */
     setSelectionRange(selection: TextRange): TextState;
+    /**
+     * Selects the specified text range
+     * @param selection
+     */
+    getState(): TextState
 }
diff --git a/node_modules/react-mde/lib/js/commandOrchestrator.js b/node_modules/react-mde/lib/js/commandOrchestrator.js
index f67979f..4822dca 100644
--- a/node_modules/react-mde/lib/js/commandOrchestrator.js
+++ b/node_modules/react-mde/lib/js/commandOrchestrator.js
@@ -15,6 +15,9 @@ var TextAreaTextApi = /** @class */ (function () {
         this.textArea.selectionEnd = selection.end;
         return getStateFromTextArea(this.textArea);
     };
+    TextAreaTextApi.prototype.getState = function () {
+        return getStateFromTextArea(this.textArea);
+    }
     return TextAreaTextApi;
 }());
 exports.TextAreaTextApi = TextAreaTextApi;
